llms/mlx_lm/LORA.md

```diff
@@ -23,54 +23,108 @@ LoRA (QLoRA).[^qlora] LoRA fine-tuning works with the following model families:
 
 ## Run
 
-The main command is `mlx_lm.lora`. To see a full list of options run:
-
-```shell
-python -m mlx_lm.lora --help
+The main command is `mlx_lm.lora`. The argument is a YAML file with the training parameters 
+in the following format:
+
+```yaml
```

Comment:
Let's keep the README as it was. Instead of putting this here, let's put an example config in the `examples` directory (call it `lora_config.yaml`). https://github.com/ml-explore/mlx-examples/tree/main/llms/mlx_lm/examples

Reply:
And maybe link to the example config in this readme and mention how to use it.